app/src/androidTest/java/it/netknights/piauthenticator/PushRolloutAuth.java

```diff
@@ -95,11 +97,13 @@
     @Before
     public void setup() throws IOException, NoSuchAlgorithmException, InvalidKeySpecException {
         server = new MockWebServer();
-
+        // Setup 'https'
+        HandshakeCertificates hsc = TlsUtil.localhost();
+        server.useHttps(hsc.sslSocketFactory(), false);
```

Comment:
*localhost* and *useHttps false* sound a bit awkward to me. I have no idea?

Reply:
the second argument is for tunnelProxy